modules/application/src/main/java/edu/northeastern/truthtree/dto/StoryDTOWrapperList.java

```diff
@@ -0,0 +1,43 @@
+package edu.northeastern.truthtree.dto;
```

Comment:
Same concerns as above. Do we need a separate DTO just for parsing while a `List<Map<String, Object>>` would suffice?

Reply:
Tried and tested this. I want to maintain order of the stories which will be lost if I keep in the map.